golang/helpers/constants.go

```diff
@@ -0,0 +1,26 @@
+package helpers
```

Comment:
not too important, but could rename to `const.go` - https://go.dev/src/math/const.go